qa/rpc-tests/hardforkdetection.py

```diff
@@ -12,15 +12,10 @@
 
 class HardForkDetectionTest(BitcoinTestFramework):
 
-    alert_filename = None  # Set by setup_network
-
     def setup_network(self):
         self.nodes = []
-        self.alert_filename = os.path.join(self.options.tmpdir, "alert.txt")
-        with open(self.alert_filename, 'w', encoding='utf8'):
-            pass  # Just open then close to create zero-length file
         self.nodes.append(start_node(0, self.options.tmpdir,
-                            ["-blockversion=2", "-alertnotify=echo %s >> \"" + self.alert_filename + "\""]))
+                            ["-blockversion=2"]))
 
     def assert_safemode_off(self):
         self.nodes[0].getbalance()
```

Comment:
```suggestion
        # if safemode is on, getbalance() will throw
        self.nodes[0].getbalance()
```

Reply:
This is a good suggestion (I didn't understand that myself), but since this PR is no longer touching this file, I think it's best to leave this change out.